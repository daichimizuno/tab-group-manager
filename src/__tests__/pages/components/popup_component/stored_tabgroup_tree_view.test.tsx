import { render } from "@testing-library/react";
import React from "react";
import renderer, { act, ReactTestRendererJSON } from "react-test-renderer";
import StoredTabGroupTreeView from "../../../../pages/components/popup_component/stored_tabgroup_tree_view";

let component;
beforeEach(() => {
  act(() => {
    component = render(<StoredTabGroupTreeView />);
  });
});

afterEach(() => {
  jest.resetAllMocks();
});

describe("PopupTabViewの試験", () => {
  describe("コンポーネントテスト", () => {
    describe("正常系", () => {
      test("スナップショットが前回の結果と一致しているかどうか", () => {
        let tree = renderer
          .create(<StoredTabGroupTreeView />)
          .toJSON() as ReactTestRendererJSON;
        expect(tree).toMatchSnapshot();
      });
    });
  });
});
