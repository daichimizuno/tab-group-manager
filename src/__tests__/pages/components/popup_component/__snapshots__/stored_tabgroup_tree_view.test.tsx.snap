// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopupTabViewの試験 コンポーネントテスト 正常系 スナップショットが前回の結果と一致しているかどうか 1`] = `
<h3>
  まだタブグループが作成されていません
</h3>
`;
